polyglot.js

```diff
@@ -54,20 +55,23 @@ class PolyGlot {
     }
 
     renderChatLog(chatlog, html, data) {
-        this.setCustomLanguages(game.settings.get("polyglot", "customLanguages"));
-        const lang_html = $(`
-        <div id="polyglot"  class="polyglot-lang-select flexrow">
-                <label>Language : </label>
-                <select name="polyglot-language">
-                </select>
-        </div>
-        `);
-        html.find("#chat-controls").after(lang_html);
-        const select = html.find(".polyglot-lang-select select");
-        select.change(e => {
-            this.lastSelection = select.val();
-        })
-        this.updateUserLanguages(html)
+        this.setCustomLanguages(game.settings.get("polyglot", "customLanguages")).then(
```

Comment:
Don't use `then` here, that's really ugly. It's better to set the function as async and use `await`.